packages/svelte/src/remix-router-svelte.ts

```diff
@@ -118,14 +129,19 @@ export function useFetcher<TData = unknown>(): Readable<
   return derived(fetcherStore, (fetcher) => {
     return {
       ...fetcher,
-      // how to get fetcherKey passed in?
-      Form,
-      key: fetcherKey,
+      Form: FetcherForm,
       submit(target: SubmitTarget, options = {}) {
-        return submitForm(router, defaultAction, target, options, fetcherKey);
+        return submitForm(
+          router,
+          defaultAction,
+          target,
+          options,
+          routeId,
+          fetcherKey
```

Comment:
super nitpick, but the reversed ordering of `routeId`/`fetcherKey` here versus `router.fetch` feels like it's asking for inadvertent bugs, especially since they're both strings so TS won't be much help either